### DISCUSSION TITLE
How to apply tailwind class is work in my filament page

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#11690] [General]
Hi I use to make filament custom page in in this page I use live wire 
```
<x-filament-panels::page>
@livewire('pos')
</x-filament-panels::page>
```

and in the pos.blade in live wire, I use tailwind css but most class is not work in this blade like bg, grid and other class how to make all of tailwind class is use in this blade

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme
?
